### PR TITLE
Support gefs C48 on Azure

### LINKED ISSUE #2882: Support gefs C48 on Azure
### What new functionality do you need?

Support global-workflow GEFS C48 on Azure

### What are the requirements for the new functionality?

Make changes to Azure env and yaml files so gefs can run.

### Acceptance Criteria

All on-prem tests and runs still work properly, and gefs case run succeed on Azure

### Suggest a solution (optional)

Have already checked on EPIC branch, need file PR and test to merge back to EMC repo.

### PR DESCRIPTION
**Pull request by weihuang-jedi**
Support global-worflow GEFS C48 on Azure

# Description

Make env. var. and yaml file changes, so global-workflow GEFS C48 case can run properly on Google Cloud.

Resolves https://github.com/NOAA-EMC/global-workflow/issues/2882

# Type of change

- New feature (adds functionality)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? YES
- Does this change require a documentation update? YES
- Does this change require an update to any of the following submodules? NO (If YES, please add a link to any PRs that are pending.)
  - [ ] EMC verif-global
  - [ ] GDAS
  - [ ] GFS-utils
  - [ ] GSI
  - [ ] GSI-monitor
  - [ ] GSI-utils
  - [ ] UFS-utils
  - [ ] UFS-weather-model
  - [ ] wxflow

# How has this been tested?

Example:
- Clone and build on Azure
- GEFS C48 test on Azure

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: env/AZUREPW.env [hunk 1]
```diff
@@ -27,7 +27,7 @@ if [[ -n "${ntasks:-}" && -n "${max_tasks_per_node:-}" && -n "${tasks_per_node:-
     NTHREADS1=${threads_per_task:-1}
     [[ ${NTHREADSmax} -gt ${max_threads_per_task} ]] && NTHREADSmax=${max_threads_per_task}
     [[ ${NTHREADS1} -gt ${max_threads_per_task} ]] && NTHREADS1=${max_threads_per_task}
-    APRUN="${launcher} -n ${ntasks}"
+    export APRUN="${launcher} -n ${ntasks}"
```
Comment (aerorahul) [inline env/AZUREPW.env:30]: Do not export this variable here.  It should only be exported for a specific task.

**Review by aerorahul — CHANGES_REQUESTED** [review]
everything except 1 thing looks good.

File: env/AZUREPW.env (hunk 1 above)
Comment (aerorahul) [inline env/AZUREPW.env]:
```suggestion
   APRUN="${launcher} -n ${ntasks}"
```
do not export `APRUN` here.

**Review by aerorahul — DISMISSED** [review]
looks good

**Comment by aerorahul** [comment]
@weihuang-jedi The PR looks good, but it appears has some conflicts.  If you resolve these, we will merge this.

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Review by aerorahul — APPROVED** [review]
looks good.

**Pull request merged**